site/src/component/SearchModule/SearchModule.tsx

```diff
@@ -22,7 +22,7 @@ interface SearchModuleProps {
 const SearchModule: FC<SearchModuleProps> = ({ index }) => {
   const dispatch = useAppDispatch();
   const search = useAppSelector((state) => state.search[index]);
-  const [pendingRequest, setPendingRequest] = useState<NodeJS.Timeout | null>(null);
+  const [pendingRequest, setPendingRequest] = useState<number | null>(null);
```

Comment:
NodeJS.Timeout seems more clear from a readability standpoint so that I'd know what to do with this, but I'm fine changing to number for consistency's sake

Reply:
Timeout type/NodeJS namespace no longer exists when using pnpm